{"uuid":"621c7ca7-e16b-465b-b6b8-a2b98cd487db","name":"Scenario: Checking error message remove button","children":["e438cf7c-94d4-4769-8997-983f50577445"],"befores":[{"name":"StepDefs.Hooks.beforeScenario()","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1745148318452,"stop":1745148319892}],"afters":[{"name":"StepDefs.Hooks.afterScenario()","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER is failed: The following asserts failed:\n\texpected [2] but found [1]","trace":"java.lang.AssertionError: The following asserts failed:\n\texpected [2] but found [1]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat StepDefs.Hooks.afterScenario(Hooks.java:28)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1745148353280,"stop":1745148353590}],"start":1745148318452,"stop":1745148353590}